libraries/botbuilder-dialogs/src/prompts/choicePrompt.ts

```diff
@@ -62,25 +57,44 @@ export class ChoicePrompt extends Prompt<FoundChoice> {
      * @param dialogId Unique ID of the dialog within its parent `DialogSet`.
      * @param validator (Optional) validator that will be called each time the user responds to the prompt. If the validator replies with a message no additional retry prompt will be sent.
      * @param defaultLocale (Optional) locale to use if `dc.context.activity.locale` not specified. Defaults to a value of `en-us`.
+     * @param choiceDefaults (Optional) Overrides the dictionary of Bot Framework SDK-supported _choiceDefaults (for prompt localization).
+     *  Must be passed in to each ConfirmPrompt that needs the custom choice defaults.
      */
-    constructor(dialogId: string, validator?: PromptValidator<FoundChoice>, defaultLocale?: string) {
+    public constructor(dialogId: string, validator?: PromptValidator<FoundChoice>, defaultLocale?: string);
+
+    public constructor(dialogId: string, validator?: PromptValidator<FoundChoice>, defaultLocale?: string, choiceDefaults?: ChoiceDefaultsChoicePrompt) {
```

Comment:
You typically need a 3'rd signature when overriding like this because the base signature gets hidden by typescript. Are you sure you're able to pass in the choiceDefaults?

Reply:
@stevenic Are you saying I'd need, exactly: `public constructor()` as the first ctor?

console.logged them during the test:

```cmd
{
  "custom-custom": {
    "choices": [
      "customYes",
      "customNo"
    ],
    "options": {
      "inlineSeparator": "customSeparator",
      "inlineOr": " customOr ",
      "inlineOrMore": " customOrMore ",
      "includeNumbers": true
    }
  }
}
confirmPrompt.ts:84
    √ should accept and recognize custom locale dict
```

Reply:
removed overrides